lib/build.api.js

```diff
@@ -70,7 +69,7 @@ class Build {
       let project = this.aquifer.project;
 
       // Set some reasonable defaults.
-      this.options.makeFile = this.getAbsolutePath(this.options.makeFile) || project.absolutePaths.makeFile;
+      this.options.makeFile = project.absolutePaths.make;
```

Comment:
I think the problem with this when I originally tested it, was that is was always evaluating to true.
```
this.options.makeFile = this.options.makeFile ? this.getAbsolutePath(this.options.makeFile) : project.absolutePaths.makeFile;
```
Granted I didn't set it to false as in your suggestion above which should work.  Is it possible to overwrite this as a command line option without making other modifications to pass this data through?